src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,85 @@
+import cn from 'classnames';
+import { useContext } from 'react';
+import { Status } from '../types/Status';
+import { TodoContext } from './TodoContext';
+
+export const TodoFooter: React.FC = () => {
+  const {
+    todos, filter, setFilter, deleteTodo,
+  } = useContext(TodoContext);
+
+  const activeTodoCount = todos.filter(todo => !todo.completed).length;
+  const activeTodoCountMsg
+    = `${activeTodoCount} ${activeTodoCount === 1 ? 'item' : 'items'} left`;
+
+  const hasCompletedTodos = todos.some(todo => todo.completed);
+
+  const handleClearCompleted = () => {
+    // setIsLoading(true);
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments) Fix all cases